sql/moz-fx-data-shared-prod/telemetry_derived/firefox_android_clients_v1/init.sql

```diff
@@ -0,0 +1,312 @@
+-- Initialization query first observations for Firefox Android Clients.
+CREATE OR REPLACE TABLE
+  `moz-fx-data-shared-prod.telemetry_derived.firefox_android_clients_v1`(
+    client_id STRING NOT NULL
+    OPTIONS
+      (description = "Unique ID for the client installation."),
```

Comment:
I know you wanted to keep the descriptions so you have the schema here, but they're also down below in the `schema.yaml`. Does that file not propagate descriptions to BQ?

Reply:
That's a good point, and I'm thinking about removing the schema to avoid confusion becuase a) the process to create or re-create this table is to run the init script, so the schema.yml would not be used and b) Because of the structure of the query, running `bqetl schema update` doesn't work so this file needs to be maintained manually. Probably safer to remove it. What do you think?

Reply:
It does seem like we only need one or the other. In the bqetl paradigm we would probably opt to drop the `init.sql`, recreate the table from the `schema.yaml`, and backfill the data using `query.sql` one day at a time. That is difficult and slow, however, so if BQ is capable of loading in one go I like using `init.sql` to recreate all at once and dropping `schema.yaml`.